### PR TITLE
Release 1 8 0 helm chart

### PR DESCRIPTION
**Pull request by crasu**
## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

**Review by sklgromek — APPROVED** [review]

File: lte/gateway/deploy/agw_install_docker.sh [hunk 1]
```diff
@@ -118,4 +118,14 @@ else
...
+# check if we are on ARM system
+if [ "$(uname -m)" == "aarch64" ]; then
+  sed -i 's/OPTIONAL_ARCH_POSTFIX=/OPTIONAL_ARCH_POSTFIX=_arm/' /var/opt/magma/docker/.env
+fi
+
+if [ "${MAGMA_VERSION}" == "v1.8" ]; then
+  sed -i 's,DOCKER_REGISTRY=,DOCKER_REGISTRY=docker.artifactory.magmacore.org/,' /var/opt/magma/docker/.env
+  sed -i 's/IMAGE_VERSION=latest/IMAGE_VERSION=1.8.0/' /var/opt/magma/docker/.env
+fi
+
```
Comment (ShubhamTatvamasi) [inline lte/gateway/deploy/agw_install_docker.sh:130]: We don't need this now. We have already done change on Ansible code for making these changes.

File: orc8r/cloud/deploy/terraform/orc8r-aws/eks.tf [hunk 2]
```diff
@@ -30,13 +30,13 @@ resource "aws_key_pair" "eks_workers" {
 
 module "eks" {
   source  = "terraform-aws-modules/eks/aws"
-  version = "~> 17.0.3"
+  version = "~> 18.27.0"
```
Comment (ShubhamTatvamasi) [inline orc8r/cloud/deploy/terraform/orc8r-aws/eks.tf:33]: Have you tried deploying this module version with the latest terraform version?

File: orc8r/cloud/deploy/terraform/orc8r-aws/variables.tf [hunk 3]
```diff
@@ -105,17 +105,17 @@ variable "eks_worker_groups" {
...
-      asg_max_size         = 3
-      autoscaling_enabled  = false
-      kubelet_extra_args   = "" // object types must be identical (see thanos_worker_groups)
-    },
-  ]
+  default = {
+    eks_worker_group = {
+      name                = "wg-1"
+      instance_type       = "t3a.large"
+      desired_size        = 3
```
Comment (ShubhamTatvamasi) [inline orc8r/cloud/deploy/terraform/orc8r-aws/variables.tf:112]: Are we getting 3 desired nodes when we deploy EKS cluster?

File: orc8r/cloud/deploy/terraform/orc8r-helm-aws/examples/basic/main.tf [hunk 4]
```diff
@@ -12,7 +12,7 @@
 ################################################################################
 
 locals {
-  region = "us-west-2"
+  region = "ap-south-1"
```
Comment (ShubhamTatvamasi) [inline orc8r/cloud/deploy/terraform/orc8r-helm-aws/examples/basic/main.tf:15]: Do you think we should revert this back to `us-west-2` or it's fine?

File: orc8r/cloud/deploy/terraform/orc8r-helm-aws/examples/basic/main.tf [hunk 5]
```diff
@@ -24,28 +24,30 @@ module "orc8r" {
...
-  elasticsearch_ebs_volume_size = 32
-  elasticsearch_ebs_volume_type = "gp2"
+  # elasticsearch_domain_name     = "orc8r-es"
+  # elasticsearch_version         = "7.7"
+  # elasticsearch_instance_type   = "t2.medium.elasticsearch"
+  # elasticsearch_instance_count  = 2
+  # elasticsearch_az_count        = 2
+  # elasticsearch_ebs_enabled     = true
+  # elasticsearch_ebs_volume_size = 32
+  # elasticsearch_ebs_volume_type = "gp2"
```
Comment (ShubhamTatvamasi) [inline orc8r/cloud/deploy/terraform/orc8r-helm-aws/examples/basic/main.tf:48]: We can delete these if all the values are updated on the variables file.

File: orc8r/cloud/deploy/terraform/orc8r-helm-aws/examples/basic/main.tf [hunk 6]
```diff
@@ -24,28 +24,30 @@ module "orc8r" {
...
+
+  deploy_elasticsearch_service_linked_role = false
```
Comment (ShubhamTatvamasi) [inline orc8r/cloud/deploy/terraform/orc8r-helm-aws/examples/basic/main.tf:50]: We only make it `false` if we are deploying multiple Orc8r on the same AWS account. For the first time deployment this should be `true` which is the default value.

File: orc8r/cloud/deploy/terraform/orc8r-helm-aws/examples/basic/main.tf [hunk 7]
```diff
@@ -76,12 +78,12 @@ module "orc8r-app" {
   orc8r_db_pass    = module.orc8r.orc8r_db_pass
 
   # Note that this can be any container registry provider
-  docker_registry = "docker.artifactory.magmacore.org"
+  docker_registry = "shubhamtatvamasi"
```
Comment (ShubhamTatvamasi) [inline orc8r/cloud/deploy/terraform/orc8r-helm-aws/examples/basic/main.tf:81]: We can not use this in production. This is a personal account.

File: orc8r/cloud/deploy/terraform/orc8r-helm-aws/examples/basic/main.tf [hunk 8]
```diff
@@ -76,12 +78,12 @@ module "orc8r-app" {
...
   docker_user     = ""
   docker_pass     = ""
 
   # Note that this can be any Helm chart repo provider
-  helm_repo      = "https://artifactory.magmacore.org/artifactory/helm"
+  helm_repo      = "https://shubhamtatvamasi.github.io/magma-charts-8-8-2022"
```
Comment (ShubhamTatvamasi) [inline orc8r/cloud/deploy/terraform/orc8r-helm-aws/examples/basic/main.tf:86]: We can not use this in production. This is a personal account.

File: orc8r/cloud/deploy/terraform/orc8r-helm-aws/examples/basic/main.tf [hunk 9]
```diff
@@ -93,7 +95,7 @@ module "orc8r-app" {
   elasticsearch_disk_threshold = tonumber(module.orc8r.es_volume_size * 75 / 100)
 
   orc8r_deployment_type = "fwa"
-  orc8r_tag             = "1.7.0"
+  orc8r_tag             = "d6ba3ee"
```
Comment (ShubhamTatvamasi) [inline orc8r/cloud/deploy/terraform/orc8r-helm-aws/examples/basic/main.tf:98]: We can not use this in production. This is a personal account.

**Comment by crasu** [comment]
Converted to draft as we can use the old tf providers and only need to update versions as it seems. https://github.com/magma/magma/pull/13731 was opend for the version upgrades.

**Pull request closed without merging**